sql/procedure/auto_audience_segmentation_inference_preparation.sqlx

```diff
@@ -0,0 +1,34 @@
+  -- Copyright 2023 Google LLC
+  --
+  -- Licensed under the Apache License, Version 2.0 (the "License");
+  -- you may not use this file except in compliance with the License.
+  -- You may obtain a copy of the License at
+  --
+  --     http://www.apache.org/licenses/LICENSE-2.0
+  --
+  -- Unless required by applicable law or agreed to in writing, software
+  -- distributed under the License is distributed on an "AS IS" BASIS,
+  -- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  -- See the License for the specific language governing permissions and
+  -- limitations under the License.
+  -- Setting procedure to lookback from the day before `inference_date`
+
+DECLARE lookback_days INT64 DEFAULT 15;
+SET inference_date = DATE_SUB(inference_date, INTERVAL 1 DAY);
+
+CALL `{{project_id}}.{{dataset}}.create_dataset`(inference_date, inference_date, lookback_days);
```

Comment:
Please use a more intuitive name for the training table. Something like `audience_segmentation_training_15`

Reply:
The output here generates a table called `v_auto_audience_segmentation_inference_15`, dataset is just an intermediate table that gets deleted at the end of the procedure.